docs/api/alerting/update_rule.asciidoc

```diff
@@ -118,31 +153,43 @@ The API returns the following:
 --------------------------------------------------
 {
   "id": "ac4e6b90-6be7-11eb-ba0d-9b1c1f912d74",
-  "notify_when": "onActionGroupChange",
-  "params": {
-    "aggType": "avg",
-  },
   "consumer": "alerts",
-  "rule_type_id": "test.rule.type",
-  "schedule": {
-    "interval": "1m"
-  },
-  "actions": [],
   "tags": [],
   "name": "new name",
   "enabled": true,
   "throttle": null,
+  "schedule": {
+    "interval": "1m"
+  },
+  "params": {
+    "index": [".updated-index"],
+    "timeField": "@timestamp",
+    "groupBy": "top",
+    "aggType": "avg",
+    "timeWindowSize": 5,
+    "timeWindowUnit": "m",
+    "thresholdComparator": ">",
+    "threshold": [1000],
+    "aggField": "sheet.version",
+    "termField": "name.keyword",
+    "termSize": 6
+  },
   "api_key_owner": "elastic",
   "created_by": "elastic",
   "updated_by": "elastic",
+  "snooze_schedule": [],
```

Comment:
@XavierM Is snooze_schedule meant to be returned in the public API?

Reply:
no, it should be internal

Reply:
@Zacqary can you check what happen here?

Reply:
I recreated just now on a recent 8.4.0 build to be certain.

Reply:
In the shift from `snoozeEndTime` to `snoozeSchedule` in the API I remember having to mess with what properties were marked as internal in order to figure out why some tests were failing. Probably related to that shuffle. We can fix.

Reply:
I've created https://github.com/elastic/kibana/issues/134500

Reply:
Is it okay to merge this PR as-is (since it reflects the current behaviour) then clean up the examples in the "create" and "update" API docs when https://github.com/elastic/kibana/issues/134500 is fixed?

Reply:
I would remove `snooze_schedule` since it's a bug that it is showing up in the Update API response and if we start documenting it, some user might find it :)

Reply:
Removed in b9075b3fac96acf0b4f8f4ca97c40bb814cf92d5